scripts/install-snapshot.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash -e
+
+SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"master"}
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
`$(dirname "${BASH_SOURCE[0]}")` should be sufficient, or rather `$(dirname ${0})`